images/prometheus-postgres-exporter/image.yaml

```diff
@@ -0,0 +1,5 @@
+versions:
```

Comment:
`status: experimental`, or otherwise update READMEs to reflect

Reply:
Done, thanks!